packages/tkg-clusterclass-vsphere/bundle/config/upstream/base.yaml

```diff
@@ -541,7 +534,7 @@ spec:
       - op: replace
         path: /spec/template/spec/template
         valueFrom:
-          template: '{{ (index .TKR_DATA .builtin.controlPlane.version).osImageRef.template }}'
+          template: '/dc0/vm/photon-3-kube- {{- (index .TKR_DATA .builtin.controlPlane.version).osImageRef.version }}'
```

Comment:
We will want to revert this change before committing. This is necessary until this #2748 is merged.

Reply:
I think we should split this large PR into several small ones. At least those addon code changes should be in a different one. Could you create jira stories or bugs for your code changes so we can break them down and assign them?